test/dss_staging_sub_ids.tsv

```diff
@@ -0,0 +1,2 @@
+tenx_sub_id	ss2_sub_id
+3e3e176b-629f-46ea-b01d-e36bd650dc54	dd17bb03-7634-47a4-9fd3-a55580ac3b1c
```

Comment:
This replaces the old dev_config.json file. It's easier to pull stuff out of a tsv in bash.